tests/nwchem_tests/test_tce_qcisd.py

```diff
@@ -0,0 +1,40 @@
+#QCISD on water with sto-3g
+
+import os
+import sys
+import qcdb
+from ..addons import *
+from ..utils import *
+
+h2o = qcdb.set_molecule('''
+    H    0.000000000000000   1.079252144093028   1.474611055780858
+    O    0.000000000000000   0.000000000000000   0.000000000000000
+    H    0.000000000000000   1.079252144093028  -1.47461105578085
+       units au ''')
```

Comment:
move this `h2o = ` line into the `def test_1_qcisd` function so it's self contained and won't get confused by other such `h2o` definitions in a general `pytest tests/` run.